src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -73,7 +71,7 @@ public SwerveSubsystem(SwerveDrivetrainConstants driveTrainConstants, SwerveModu
     @Override
     public void periodic() {
         Pose2d pose = m_odometry.getEstimatedPosition();
-        SmartDashboard.putNumberArray("pose estimation", new Double[]{pose.getX(), pose.getY(), pose.getRotation().getDegrees()});
+        SmartDashboard.putNumberArray("pose raw", new Double[]{pose.getX(), pose.getY(), m_pigeon2.getRotation2d().getRadians()});
```

Comment:
Any reason to use radians over degrees? Doesn't matter much to me, just figured y'all would be more used to degrees.